test/Sentry.Tests/ApiApprovalTests.Run.Core3_1.verified.txt

```diff
@@ -1623,6 +1635,7 @@ namespace Sentry.Protocol
         public Response() { }
         public long? BodySize { get; set; }
         public string? Cookies { get; set; }
+        public object? Data { get; set; }
```

Comment:
@bitsandfoxes let me know if we can do this...

Reply:
I'm quoting @bruno-garcia here: Adding optional fields to a class is not a breaking change. Adding it to an interface would be.